CRM/Member/BAO/Query.php

```diff
@@ -312,6 +312,18 @@ public static function whereClauseSingle(&$values, &$query) {
         }
         $query->_tables['civicrm_membership'] = $query->_whereTables['civicrm_membership'] = 1;
         return;
+
+      case 'member_override':
```

Comment:
I think we can standarize Qill too by using "[Label] [value]" format.
e.g "[Is membership override?] [Yes]" for membership.is_override = 1 
e.g. "[Is contribution test?] [No]" for contribution.is_test = 0

And for whereClause build
$query->_where[$grouping][] = CRM_Contact_BAO_Query::buildClause("civicrm_membership.is_override", $op, $value, "Boolean");